modules/Rock.cmake

```diff
@@ -984,13 +983,32 @@ function(rock_gtest TARGET_NAME)
     rock_add_test(${TARGET_NAME} "${__rock_test_parameters}")
 endfunction()
 
+function(rock_get_clang_targets VAR filepath)
+    if (EXISTS "${filepath}")
+        file(STRINGS ${filepath} checklist REGEX "^(\\+|-)+")
+        foreach(entry ${checklist})
+            string(REGEX MATCH "^(\\+|-)?" symbol ${entry})
+            string(REPLACE ${symbol} "" filename ${entry})
```

Comment:
This will remove a + or - within the file name (`-` is in particular not unheard of). Just strip the first character.

Also, I think glob_pattern would be better than filename as a variable name.